Mathlib/Algebra/FreeMonoid/Basic.lean

```diff
@@ -407,4 +415,31 @@ theorem length_reverse {a : FreeMonoid α} : a.reverse.length = a.length :=
 
 end Reverse
 
+section IsomorphicTypes
+
+variable {α β : Type*} (e : α ≃ β)
+
+/-- free monoids over isomorphic types are isomorphic -/
+@[to_additive "if two types are isomorphic, the additive free monoids over those types are
+isomorphic"]
+def congr_iso : FreeMonoid α ≃* FreeMonoid β :=
```

Comment:
We use "iso" for the category-theoretic notion, and "equiv" for rest.
```suggestion
def congrEquiv : FreeMonoid α ≃* FreeMonoid β :=
```

Reply:
got it, thanks!